app/controllers/events/view/export.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+
+const { Controller, run, $ } = Ember;
+export default Controller.extend({
+  eventExportStatus : '',
+  eventDownloadUrl  : '',
+  actions           : {
+    startGeneration() {
+      let payload = {
+        image    : this.get('dataImage'),
+        video    : this.get('dataVideo'),
+        audio    : this.get('dataAudio'),
+        document : this.get('dataDocument')
+      };
+      let exportUrl = `/events/${this.get('model').id}/export/json`;
+      this.get('loader')
+        .post(exportUrl, payload)
+        .then(data => {
+          let requestLoop = run.later(() => {
```

Comment:
Did you test this code ? Did you read the docs on run.later ? Run.later is analogous to `setTimeout`. It will run only once after a given delay. You should modify your code accordingly to make it work like `setInterval`